### PR TITLE
CP-41983: Update iperf2 to 3

### PR DESCRIPTION
**Pull request by tianxiahcp**

### CODE REVIEW

**Comment by lindig** [comment]
This needs a better commit message including a sign-off.

File: setup-scripts/init-run.sh [hunk 1]
```diff
@@ -10,8 +10,9 @@ dnf --enablerepo=powertools install perl-Readonly -y
 # setup firewall port for 4/tcp/udp and 5001/tcp/udp
 firewall-cmd --zone=public --add-port=4/tcp --permanent
 firewall-cmd --zone=public --add-port=4/udp --permanent
-firewall-cmd --zone=public --add-port=5001/tcp --permanent
-firewall-cmd --zone=public --add-port=5001/udp --permanent
+# open port for iperf3
```
Comment (lindig) [inline setup-scripts/init-run.sh:13]: Port numbers change. This should go into the commit message and probably into documentation. Does this need discussion?

**Review by lindig — CHANGES_REQUESTED** [review]

File: autocertkit/network_tests.py [hunk 2]
```diff
@@ -267,31 +267,22 @@ def run_iperf_server(self):
...
         rec = {}
-        rec['datetime'] = arr[0]
-        rec['client_ip'] = arr[1]
-        rec['client_port'] = arr[2]
-        rec['server_ip'] = arr[3]
-        rec['server_port'] = arr[4]
-        rec['id'] = arr[5]
-        rec['interval'] = arr[6]
-        rec['transfer'] = arr[7]
-        rec['bandwidth'] = arr[8]
```
Comment (DeliZhangX) [inline autocertkit/network_tests.py:288]: `bandwidth` is useful, should keep it and I saw iperf3 still support it from https://iperf.fr/iperf-doc.php

File: autocertkit/network_tests.py [hunk 3]
```diff
@@ -317,7 +308,7 @@ def copy(param, arg_str):
             test_ip = self.vm_info[self.server]['ip_t']
             protocol = ''
 
-        cmd_str = "iperf -y csv %s %s -m -B %s -c %s" % \
+        cmd_str = "iperf -J %s %s -B %s -c %s" % \
```
Comment (DeliZhangX) [inline autocertkit/network_tests.py:311]: Should consider to run `iperf3` if applied.

**Comment by DeliZhangX** [comment]
@catherineliuchina You could submit new PR with fixes as Tian Xia left XS.

**Pull request closed without merging**